aria/workflow_engine/tasks.py

```diff
@@ -128,19 +128,27 @@ class StartWorkflowTask(EngineBaseTask):
 
     def __init__(self, *args, **kwargs):
         super(StartWorkflowTask, self).__init__(*args, **kwargs)
-        self.on_success_signal.connect(self._create_execution_in_starage)
+        self.start_signal.connect(self._create_execution_in_storage)
 
     def complete(self):
-        execution = self.context.storage.execution.get(self.context.execution_id)
-        return all([
-            execution.status == Execution.STARTED,
-            super(StartWorkflowTask, self).complete(),
-        ])
-
-    def _create_execution_in_starage(self, **kwargs):
-        # todo: maybe to change name? how will create the model in the data base?
-        execution = self.context.storage.execution.get(self.context.execution_id)
-        execution.status = Execution.STARTED
+        try:
+            self.context.storage.execution.get(self.context.execution_id)
+        except StorageError:
+            return False
+        return True
```

Comment:
you really can't, super of StartWorkflowTask have no complete, so it reverts to the base complete, which checks for outputs, and returns True only if all outputs are available.